docker/Dockerfile-fedora

```diff
@@ -0,0 +1,28 @@
+from fedora:latest
+
+ARG APP
+ARG DEPS
+ARG INSECURE
+
+RUN yum update -y
+RUN yum install -y make gcc cmake git mercurial graphviz gnupg2 R perl perl-devel openmpi openmpi-devel
+
+RUN useradd -m ci
+
+COPY proxy-ci/ /home/ci/proxy-ci
+COPY spack /home/ci/spack
+RUN rm -rf /home/ci/.ccache
+COPY ccache/ /home/ci/.ccache
+USER root
+RUN chown -R ci:ci /home/ci/proxy-ci /home/ci/.ccache /home/ci/spack
+USER ci
+
+WORKDIR /home/ci
+RUN cp proxy-ci/fedora-packages.yaml spack/etc/spack/packages.yaml
+ENV SPACK=/home/ci/spack/bin/spack
+ENV SPACK_ROOT=/home/ci/spack
+
+RUN source ${SPACK_ROOT}/share/spack/setup-env.sh
+RUN ${SPACK} install -j4 patch
```

Comment:
Huh? Why?

Reply:
That is related to ongoing issues with debugging what is different between the docker and interactive environments for the Fedora container with a subset of the proxies